imap_processing/cdf/config/imap_hi_variable_attrs.yaml

```diff
@@ -91,7 +91,7 @@ hi_de_esa_stepping_num:
   <<: *hi_esa_stepping_num
 
 hi_de_event_met:
-  <<: *default_uint64
+  <<: *default_int64
```

Comment:
Do any of these need to be `uint32`? I think that MET has typically been 32 bit unsigned so wondering if this should be the same as that.

Reply:
I believe `Hi` needs int64 because `Hi` converts `met_time` to nanosecond accuracy. https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/d24478fafa7f477a8fb394945057baef78ffb7c5/imap_processing/hi/l1a/science_direct_event.py#L243

Reply:
Correct. The Direct Event MET is in nanoseconds. Also in L1A is the CCSDS MET which is a uint32 and stores seconds.